### ISSUE TITLE
v1.12.0 Fails to build

### ISSUE THREAD

**Issue by nickfedoroff** [one-signal/OneSignal-iOS-SDK#23]
After installing 1.12.0 via Cocoapods, the OneSignal build fails with many errors. See attached screenshot.
![screen shot 2016-03-29 at 5 12 16 pm](https://cloud.githubusercontent.com/assets/1442451/14126908/6fc932e0-f5d1-11e5-889b-042d4fc20722.png)

**Comment by jkasten2** [comment]
Hello,

Sorry about the issue, we removed `Foundation.h` from our header as it did not create issues in our testing when using it in the AppDelegate class. We will add it back in an update but until then you can work around the issue by adding the following header before importing `OneSignal.h`. Example.
```Objective-c
#import <Foundation/Foundation.h>
#import <OneSignal/OneSignal.h>
```

Thanks.

**Comment by nickfedoroff** [comment]
Will do. Thanks for the quick response.

**Comment by jkasten2** [comment]
Just fixed this with our 1.12.1 release. Just push it Cocoapods as well so it should be showing up soon.

Thanks.
